cargo-dist/tests/integration-tests.rs

```diff
@@ -77,6 +77,27 @@ scope = "@axodotdev"
     })
 }
 
+#[test]
+fn akaikatana_basic() -> Result<(), miette::Report> {
+    let test_name = _function_name!();
+    AKAIKATANA_REPACK.run_test(|ctx| {
+        let dist_version = ctx.tools.cargo_dist.version().unwrap();
+
+        let results = ctx.cargo_dist_build_global(test_name, format!(r#"
+[workspace.metadata.dist]
+cargo-dist-version = "{dist_version}"
+rust-toolchain-version = "1.67.1"
+ci = ["github"]
+installers = ["shell", "powershell"]
+targets = ["x86_64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-pc-windows-msvc", "aarch64-apple-darwin"]
+
+"#))?;
+        results.check_all(ctx, ".cargo/bin/")?;
+
+        Ok(())
+    })
+}
+
```

Comment:
And this is all you need to do to use that repo in a test (note that even though this is the vanilla `workspace.metadata.dist` we still have to overwrite it to make `cargo-dist-version` be the currently running one to avoid integrity checks). Potentially this should be automated in `patch_cargo_toml` because it's tedious for every test to do...